pages/services.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Contact from "@/components/general/Contact";
+import Head from "next/head";
+import ServicesTitle from "@/components/ServicePage/ServiceTitle";
+import Services from "@/components/ServicePage/Services";
+
+export default function service() {
+  return (
+    <>
+      <Head>
+        <title>Service page - TaoTech</title>
+        <meta name="description" content="Our services" />
```

Comment:
You did a good job for noticing the SEO. It will be better if you providing more detailed information.
![image](https://user-images.githubusercontent.com/73293255/225846838-cbd10a2a-1ed6-4dd6-8814-7916ea417aaf.png)